frameworks/Java/play2-java/play2-java-ebean-hikaricp.dockerfile

```diff
@@ -5,4 +5,4 @@ COPY play2-java-ebean-hikaricp .
 RUN sed -i 's/.enablePlugins(PlayJava, PlayEbean, PlayNettyServer)/.enablePlugins(PlayJava, PlayEbean).disablePlugins(PlayNettyServer)/g' build.sbt
 
 RUN sbt stage
-CMD ["target/universal/stage/bin/play2-java-ebean-hikaricp", "-Dplay.server.provider=play.core.server.AkkaHttpServerProvider", "-J-server", "-J-Xms1g", "-J-Xmx1g", "-J-XX:NewSize=512m", "-J-XX:+UseG1GC", "-J-XX:MaxGCPauseMillis=30", "-J-XX:-UseBiasedLocking", "-J-XX:+AlwaysPreTouch"]
+CMD target/universal/stage/bin/play2-java-ebean-hikaricp -Dplay.server.provider=play.core.server.AkkaHttpServerProvider -J-server -J-Xms1g -J-Xmx1g -J-XX:NewSize=512m -J-XX:+UseG1GC -J-XX:MaxGCPauseMillis=30 -J-XX:-UseBiasedLocking -J-XX:+AlwaysPreTouch -Dthread_count=$(nproc) -Dphysical_cpu_count=$(grep ^cpu\\scores /proc/cpuinfo | uniq | awk '{print $4}') -Ddb_pool_size=$(( $(( $(grep ^cpu\\scores /proc/cpuinfo | uniq | awk '{print $4}') * 2 )) + 1 ))
```

Comment:
Yikes

Reply:
You don't like it?

Reply:
Haha, if I didn't like it, I wouldn't have merged it... that said... yikes.

Reply:
It's a bit cumbersome, I know, but it should do the job :wink: